dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -295,6 +301,12 @@ def install_js_error_recorder
   @browser.execute_script("$(\"#{jquery_selector}\")[0].click();")
 end
 
+When /^I click selector "([^"]*)" if it exists$/ do |jquery_selector|
+  if @browser.execute_script("return (\"#{jquery_selector}\").length > 0")
```

Comment:
Well, the string length is always `> 0`.  Are we ever hitting the negative case where the thing you might click is not there?